tools/fuchsia/gen_package.py

```diff
@@ -91,20 +92,12 @@ def main():
   ]
 
   # Build the package
-  try:
-    output = subprocess.check_output(pm_command_base + ['build'])
-  except subprocess.CalledProcessError as e:
-    print('The "%s" command failed:' % e.cmd)
-    print(e.output)
-    raise
+  # Use check_output so if anything goes wrong we get the output.
+  subprocess.check_output(pm_command_base + ['build'])
```

Comment:
I'd like to have the script print the contents of the manifest if this command fails.

The error on LUCI is consistent with an empty `from_package` in the manifest.  However, that shouldn't be possible - `os.path.abspath` should return a nonempty string containing the current working directory path.  There might be something else wrong with the manifest.

Reply:
Added that